astropy/cosmology/flrw/tests/test_base.py

```diff
@@ -476,19 +480,8 @@ def test_init_Ob0(self, cosmo_cls, ba):
         with pytest.raises(ValueError, match="baryonic density can not be larger"):
             cosmo_cls(*tba.args, **tba.kwargs)
 
-        # No baryons specified means baryon-specific methods fail.
-        tba = copy.copy(ba)
-        tba.arguments.pop("Ob0", None)
-        cosmo = cosmo_cls(*tba.args, **tba.kwargs)
-        with pytest.raises(ValueError):
-            cosmo.Ob(1)
```

Comment:
Proabbly makes sense to check that `cosmo.Ob(1) == 0`

Reply:
SGTM.

Reply:
✅